roles/edpm_ovs/tasks/install.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install ovs packages
+  tags:
+    - install
+    - ovs
+  become: true
+  ansible.builtin.package:
+    name: "{{ item }}"
+    state: present
+  loop: "{{ edpm_ovs_packages }}"
+  notify:
+    - Restart ovs
+
+- name: Ensure ovs services are enabled and running
```

Comment:
may be add state: started here itself(also to match it with task name) instead of adding a "Restart" notify handler and using flush_handlers?

Reply:
so im trying to prefer using the handeler over using started in general to have a single way to restart/start the service once if we have configtion action that require the process to be bounded.

in the case of ovs most of its config is done by the ovs db so we can change some of it at run time but not all its config.

since I'm intentionally not doing any config here and derfing that to os-net-config i guess swaping to start might be fine for ovs so ill make that refactoring but for service that are config file based i think the handeler approach is cleaner.